starter-code/views/about-page.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title>About</title>
+  </head>
+  <body>
+    <nav class="navbar">
+      <ul>
+        <li><a href="http://localhost:3000/home-page">Home</a></li>
```

Comment:
This href does not work if you deploy your project as your webpage will not be named localhost:3000. What you can write instead is: 
 <li.><.a href="/home-page">Home<./a><./li>

This will bring you to whatever your page is called ..../home. eg: www.markus.com/home. or localhost:3000/home